script/travis-install-linux

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+# See: https://github.com/kylef/swiftenv/wiki/Travis-CI
+curl -sL https://gist.github.com/kylef/5c0475ff02b7c7671d2a/raw/621ef9b29bbb852fdfd2e10ed147b321d792c1e4/swiftenv-install.sh | bash
```

Comment:
It could also be set in `.travis.yml` via an environmental variable (as `SWIFT_VERSION`) if preferred (instead of the file), but the file is more generic and then swiftenv locally will automatically use the correct version (or error if not installed).